Data/ByteString/Streaming/Char8.hs

```diff
@@ -633,38 +633,52 @@ lineSplit :: forall m r. Monad m
   => Int -- ^ number of lines per group
   -> ByteString m r -- ^ stream of bytes
   -> Stream (ByteString m) m r
-lineSplit !n0 text0 = loop1 0 text0
+lineSplit !n0 text0 = loop1 text0
   where
     n :: Int
     !n = max n0 1
-    loop1 :: Int -> ByteString m r -> Stream (ByteString m) m r
-    loop1 !counter text =
+    loop1 :: ByteString m r -> Stream (ByteString m) m r
+    loop1 !text =
```

Comment:
```suggestion
    loop1 text =
```

This bang is redundant - it is immediately followed by `case text of { ... }`

Reply:
Sure.  Will drop it.

Reply:
Pushed fixup.